chef/cookbooks/hyperv/recipes/openstack_install_neutron.rb

```diff
@@ -25,3 +24,13 @@
   not_if { ::File.exists?("#{node[:openstack][:neutron][:installed]}") }
 end
 
+ruby_block "ensure correct python path in shebang" do
```

Comment:
#!C:\Python27_HyperVNovaCompute\python.exe
No idea why this path was used, but having the shebang replaced with our python path can only avoid this in the future.